src/renderer/components/editorWithTabs/tabs.vue

```diff
@@ -134,6 +134,9 @@ export default {
       bus.$on('TABS::rename', this.rename)
       bus.$on('TABS::copy-path', this.copyPath)
       bus.$on('TABS::show-in-folder', this.showInFolder)
+      bus.$on('EDITOR_TABS::change-max-width', (width) => {
+        this.$store.state.layout.sideBarWidth = width
```

Comment:
Could you please use `this.$store.dispatch('CHANGE_SIDE_BAR_WIDTH', width)` that the application can handle the width change. I think you can remove this event and just execute the event from below.

Reply:
Yeah, sure, It's look much more cleanner now, thx for the advice.

Reply:
But I don't think I can remove the event cause the new width need to be passed from a component to another. May be I'am wrong, and if I am, just correct me.